src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -6,6 +6,9 @@
 
 from vivarium_census_prl_synth_pop import utilities
 from vivarium_census_prl_synth_pop.constants import paths
+from vivarium_census_prl_synth_pop.results_processing.generate_names import (
+    generate_names,
```

Comment:
I think if we made an `__init__.py` in results_processing we could eliminate this repetition of "generate_names," which is also going to look nicer when we have multiple postprocessing steps.

Reply:
This is a good idea.  Doing this I wasn't sure how exactly we would want to set up the results processing stage since we will have multiple steps and I am not sure what would be necessary for each step to do.